metadata-ws/src/test/resources/features/reference-sequence.feature

```diff
@@ -329,24 +338,34 @@ Feature: reference sequence
 
   Scenario Outline: search various reference sequences by taxonomy name and id
```

Comment:
At this point I think it would be more clear to use real cases instead of TAXONOMY_UNRELATEDNUMBERS.